docs/ContainerRuntimeConfigDesign.md

```diff
@@ -56,7 +56,23 @@ ContainerRuntimeConfig
  [ContainerRuntimeConfigurationSpec](Will have to add this in this repo)
 ```
 
-## Example 
+## VALIDATION
+
+It's importent to note that, since the fields of the ContainerRuntimeConfig are directly read by the upstream kubernetes golang client, the validation of those values is handled directly by that golang client which is outside of the controller for ContainerRuntimeConfig. Please ensure the valid values are used for those fields as invalid values may render cluster nodes unusable.
+
+### Troubleshooting Validation Issues
+
+If the specified field values of the ContainerRuntimeConfig cannot be converted to their inteded types then errors related validation maybe be visible on the ContainerRuntimeConfig CR, this is becaues such errors occur outside the coverage of the controller responsible for ContainerRuntimeConfig. In such case, users may need to look into the logs of the `machine-config-controller` container to get more information.
```

Comment:
```suggestion
If the specified field values of the ContainerRuntimeConfig cannot be converted to their intended types, validation errors related may be visible on the ContainerRuntimeConfig CR. This is because such errors occur outside the coverage of the controller responsible for ContainerRuntimeConfig. In such case, users may need to look into the logs of the `machine-config-controller` container to get more information.
```